build/tasks/esbuild.js

```diff
@@ -0,0 +1,34 @@
+const { build } = require('esbuild');
+const path = require('path');
+
+module.exports = function(grunt) {
+	grunt.registerMultiTask(
+		'esbuild',
+		'Task to run the esbuild javascript bundler',
+		function() {
+			const done = this.async();
+			const files = grunt.task.current.data.files;
+
+			files.forEach(file => {
+				const src = Array.isArray(file.src) ? file.src : [file.src];
+				const dest = file.dest;
+
+				src.forEach(entry => {
+					const name = path.basename(entry);
+					if (file.cwd) {
+						entry = path.join(file.cwd, entry);
+					}
+
+					build({
+						entryPoints: [entry],
+						outfile: path.join(dest, name),
+						minify: false,
+						bundle: true
+					})
+						.then(done)
+						.catch(done);
```

Comment:
Should be calling `done(false)` here according to the Grunt docs

Reply:
[Error also is valid](https://gruntjs.com/inside-tasks):

> Either false or an Error object may be passed to the done function to instruct Grunt that the task has failed.

Reply:
Ah cool, ok 👍